cms/static/js/views/settings/main.js

```diff
@@ -367,12 +367,19 @@ var DetailsView = ValidatingView.extend({
             message = this.post_enrollment_email_elem.val();
         }
 
-        $.post("/settings/send_test_enrollment_email",
+        // Validation for keyword substitution
+        var validation = KeywordValidator.validate_string(message);
+        if (!validation.is_valid) {
+            message = gettext("There are invalid keywords in your email. Please check the following keywords and try again:");
```

Comment:
There are still tons of other places using double quotes, should there be a separate PR to tidy those up as well?